lib/views/item-scene.js

```diff
@@ -18,21 +21,34 @@ const {
   Linking,
 } = ReactNative;
 
+const accessToken = 'pk.eyJ1IjoiZ21hcnR5IiwiYSI6ImNpdm1lanB1azAwNm8yb3A0OXZ6bmUxM2oifQ.i2HycDMBdPoatAB-ONR6FQ';
```

Comment:
Let's put this in the `config.js` file. Maybe ask @samgiles or @arcturus about how we can protect keys. Prob not important right now though as this particular key doesn't hold much value.